### PR TITLE
Adding prometheus rules to elasticsearch.

### PR DESCRIPTION
**Pull request by nutellinoit**
Hi guys, thanks to @lzecca78 i added some prometheus rules on elasticsearch!

This PR should address the issue #4

### CODE REVIEW

**Comment by angelbarrera92** [comment]
Looks pretty cool. Can you add some docs about these alerts?

Thanks

File: katalog/elasticsearch-single/rules.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        - alert: ElasticClusterYellow
+          expr: elasticsearch_cluster_health_status{color="yellow"} == 1
+          for: 30m
+          labels:
+            severity: warning
+          annotations:
+            summary: 'Elastic Cluster Yellow (instance {{ $labels.instance }})'
+            description: "Elastic Cluster Yellow status\n  VALUE = {{ $value }}\n  LABELS: {{ $labels }}"
+        - alert: ElasticNumberOfElasticHealthyNodes
+          expr: elasticsearch_cluster_health_number_of_nodes < number_of_nodes
```
Comment (lnovara) [inline katalog/elasticsearch-single/rules.yml:31]: Where is `number_of_nodes` defined?

**Comment by lnovara** [comment]
[Kube-proxy Dashboard & Alerts](https://trello.com/c/YeFuTX4g/28-kube-proxy-dashboard-alerts)

**Comment by lnovara** [comment]
[Elasticsearch Dashboard & Alerts](https://trello.com/c/6W1g5YsF/26-elasticsearch-dashboard-alerts)

File: katalog/elasticsearch-single/rules.yml (hunk 1 above)
Comment (nutellinoit) [reply katalog/elasticsearch-single/rules.yml]: Removed alert, there was no number_of_nodes defined

**Comment by nutellinoit** [comment]
> Thank you @nutellinoit! Awesome work!
> 
> As @angelbarrera92 said, can you add some docs about the alerts?
> 
> Furthermore, the alerts' annotations are not compatible with our [default Alertmanager template](https://github.com/sighupio/fury-kubernetes-monitoring/blob/master/katalog/alertmanager-operated/alertmanager.tmpl). Can you drop the `description` annotation (since `{{ $value }}` is not recomputed every time and we already have account for labels in the default template) and change `summary` to `message`?
> 
> Thank you. 🤗

@lnovara i think i did the requested updates



> Looks pretty cool. Can you add some docs about these alerts?
> 
> Thanks

@angelbarrera92 added some docs on the alerts!

**Pull request merged**